package.json

```diff
@@ -2,35 +2,36 @@
   "name": "@openmrs/esm-primary-navigation-app",
   "version": "2.0.0",
   "description": "Main navbar microfrontend for the OpenMRS SPA",
+  "homepage": "https://github.com/openmrs/openmrs-esm-primary-navigation#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openmrs/openmrs-esm-primary-navigation.git"
+  },
+  "license": "MPL-2.0",
```

Comment:
Just because we touch this file anyway: Could we add `"keywords": ["openmrs"]`?

Reply:
Added